examples/solids/src/cloptions.c

```diff
@@ -25,11 +25,8 @@
 // Process general command line options
 PetscErrorCode ProcessCommandLineOptions(MPI_Comm comm, AppCtx appCtx) {
   PetscErrorCode ierr;
-  PetscBool degreeFalg   = PETSC_FALSE;
+  PetscBool degreeFlag   = PETSC_FALSE;
```

Comment:
Do we want to just set this to automatically use `degree = 2` like we do with some of our other examples?